src/js/controller/model.js

```diff
@@ -381,6 +388,19 @@ define([
             }
         };
 
+
+        this.setPlaybackRate = function(playbackRate) {
+            if (!_.isNumber(playbackRate) || this.get('streamType') === 'LIVE' || !_attached) {
+                return false;
+            }
+
+            // Clamp the rate between 0.25x and 4x speed
+            var clampedRate = Math.max(0.25, Math.min(playbackRate, 4));
+
+            // Providers which support changes in playback rate will return the rate that we changed to
+            this.set('playbackRate', _provider ? _provider.setPlaybackRate(clampedRate) : clampedRate);
```

Comment:
Set "defaultPlaybackRate". Only set "playbackRate" based on default setting when a provider is available or when reattaching the provider.